srcpkgs/postgresql12/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'postgresql12'
 pkgname=postgresql12
 version=12.3
-revision=1
+revision=5
```

Comment:
Why revbump to 5?

Reply:
I did few rebuilds locally. Will reset when finished.